services/web-app/components/catalog-item/catalog-item-meta/catalog-item-meta.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Scales } from '@carbon/react/icons'
+import clsx from 'clsx'
+import PropTypes from 'prop-types'
+import { assetPropTypes } from 'types'
+
+import StatusIcon from '@/components/status-icon'
+import { status } from '@/data/status'
+import { getLicense } from '@/utils/schema'
+
+import styles from './catalog-item-meta.module.scss'
+
+const CatalogItemMeta = ({ asset, className, properties }) => {
+  const renderStatus = () => {
+    const statusKey = asset?.content.status?.key ?? asset?.content.status ?? 'draft'
```

Comment:
It seems like this implies that the status key can either come in on the `status` field or on the `status.key` field. Is there any way to adjust that so the key is always present on the same field in the asset?

Reply:
yeah so this is how we have it defined in our schema right now (see comment on: https://carbon-design-system.github.io/carbon-platform/resource-schemas.html#asset-status). We can open up a ticket to standardize at the `getLibraryAssets()` level if desired but that'd fall out of the scope of this PR; I'm just moving this out of `services/web-app/components/catalog-item/catalog-item.js` and into it's own component so I can re-use it

Reply:
Summarized in #562

Reply:
Oof, that definitely makes parsing the schema harder, but I see the value in the convenience for the end user to be able to forego the note if all they want to add is the key. Thanks for opening the issue!